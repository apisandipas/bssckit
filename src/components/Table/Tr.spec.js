import React from 'react';
import 'jest-styled-components';
import { render } from '@testing-library/react';
import Tr from './Tr';

describe('Tr component', () => {
  it('matches Tr active hover', () => {
    const { container } = render(<Tr active hover />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('matches Tr danger hover', () => {
    const { container } = render(<Tr danger hover />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('matches Tr dark hover', () => {
    const { container } = render(<Tr dark hover />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('matches Tr info hover', () => {
    const { container } = render(<Tr info hover />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('matches Tr light hover', () => {
    const { container } = render(<Tr light hover />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('matches Tr primary hover', () => {
    const { container } = render(<Tr primary hover />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('matches Tr secondary hover', () => {
    const { container } = render(<Tr secondary hover />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('matches Tr success hover', () => {
    const { container } = render(<Tr success hover />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('matches Tr warning hover', () => {
    const { container } = render(<Tr warning hover />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
