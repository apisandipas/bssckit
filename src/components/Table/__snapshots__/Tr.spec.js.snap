// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tr component matches Tr active hover 1`] = `
.c0 > th,
.c0 > td,
.c0 tbody tr:hover,
.c0 tbody tr:hover > th,
.c0 tbody tr:hover > td {
  background-color: rgba(0,0,0,0.1);
}

@media (min-width:576px) {

}

@media (min-width:768px) {

}

@media (min-width:992px) {

}

@media (min-width:1200px) {

}

<tr
  class="sc-fzpans c0"
/>
`;

exports[`Tr component matches Tr danger hover 1`] = `
.c0 > th,
.c0 > td {
  background-color: #f5b4bb;
}

.hover .c0:hover > th,
.hover .c0:hover > td,
.hover .c0 tbody tr:hover,
.hover .c0 tbody tr:hover > th,
.hover .c0 tbody tr:hover > td {
  background-color: #dc3545;
}

@media (min-width:576px) {

}

@media (min-width:768px) {

}

@media (min-width:992px) {

}

@media (min-width:1200px) {

}

<tr
  class="sc-fzpans c0"
/>
`;

exports[`Tr component matches Tr dark hover 1`] = `
.c0 > th,
.c0 > td {
  background-color: #abaeb1;
}

.hover .c0:hover > th,
.hover .c0:hover > td,
.hover .c0 tbody tr:hover,
.hover .c0 tbody tr:hover > th,
.hover .c0 tbody tr:hover > td {
  background-color: #343a40;
}

@media (min-width:576px) {

}

@media (min-width:768px) {

}

@media (min-width:992px) {

}

@media (min-width:1200px) {

}

<tr
  class="sc-fzpans c0"
/>
`;

exports[`Tr component matches Tr info hover 1`] = `
.c0 > th,
.c0 > td {
  background-color: #a3d6de;
}

.hover .c0:hover > th,
.hover .c0:hover > td,
.hover .c0 tbody tr:hover,
.hover .c0 tbody tr:hover > th,
.hover .c0 tbody tr:hover > td {
  background-color: #17a2b8;
}

@media (min-width:576px) {

}

@media (min-width:768px) {

}

@media (min-width:992px) {

}

@media (min-width:1200px) {

}

<tr
  class="sc-fzpans c0"
/>
`;

exports[`Tr component matches Tr light hover 1`] = `
.c0 > th,
.c0 > td {
  background-color: #fdfdfe;
}

.hover .c0:hover > th,
.hover .c0:hover > td,
.hover .c0 tbody tr:hover,
.hover .c0 tbody tr:hover > th,
.hover .c0 tbody tr:hover > td {
  background-color: #cccecf;
}

@media (min-width:576px) {

}

@media (min-width:768px) {

}

@media (min-width:992px) {

}

@media (min-width:1200px) {

}

<tr
  class="sc-fzpans c0"
/>
`;

exports[`Tr component matches Tr primary hover 1`] = `
.c0 > th,
.c0 > td {
  background-color: #9cc9fb;
}

.hover .c0:hover > th,
.hover .c0:hover > td,
.hover .c0 tbody tr:hover,
.hover .c0 tbody tr:hover > th,
.hover .c0 tbody tr:hover > td {
  background-color: #007bff;
}

@media (min-width:576px) {

}

@media (min-width:768px) {

}

@media (min-width:992px) {

}

@media (min-width:1200px) {

}

<tr
  class="sc-fzpans c0"
/>
`;

exports[`Tr component matches Tr secondary hover 1`] = `
.c0 > th,
.c0 > td {
  background-color: #b4b5b7;
}

.hover .c0:hover > th,
.hover .c0:hover > td,
.hover .c0 tbody tr:hover,
.hover .c0 tbody tr:hover > th,
.hover .c0 tbody tr:hover > td {
  background-color: #6c757d;
}

@media (min-width:576px) {

}

@media (min-width:768px) {

}

@media (min-width:992px) {

}

@media (min-width:1200px) {

}

<tr
  class="sc-fzpans c0"
/>
`;

exports[`Tr component matches Tr success hover 1`] = `
.c0 > th,
.c0 > td {
  background-color: #72d889;
}

.hover .c0:hover > th,
.hover .c0:hover > td,
.hover .c0 tbody tr:hover,
.hover .c0 tbody tr:hover > th,
.hover .c0 tbody tr:hover > td {
  background-color: #28a745;
}

@media (min-width:576px) {

}

@media (min-width:768px) {

}

@media (min-width:992px) {

}

@media (min-width:1200px) {

}

<tr
  class="sc-fzpans c0"
/>
`;

exports[`Tr component matches Tr warning hover 1`] = `
.c0 > th,
.c0 > td {
  background-color: #f3da8d;
}

.hover .c0:hover > th,
.hover .c0:hover > td,
.hover .c0 tbody tr:hover,
.hover .c0 tbody tr:hover > th,
.hover .c0 tbody tr:hover > td {
  background-color: #ffc107;
}

@media (min-width:576px) {

}

@media (min-width:768px) {

}

@media (min-width:992px) {

}

@media (min-width:1200px) {

}

<tr
  class="sc-fzpans c0"
/>
`;
